src/main/java/com/commercetools/sync/customers/helpers/CustomerBatchValidator.java

```diff
@@ -0,0 +1,223 @@
+package com.commercetools.sync.customers.helpers;
+
+import com.commercetools.sync.commons.helpers.BaseBatchValidator;
+import com.commercetools.sync.commons.helpers.BaseSyncStatistics;
+import com.commercetools.sync.customers.CustomerSyncOptions;
+import io.sphere.sdk.customers.CustomerDraft;
+import io.sphere.sdk.models.Address;
+import org.apache.commons.lang3.tuple.ImmutablePair;
```

Comment:
Better to follow alphabetical order for package import?

Reply:
I did not get this comment, what do you mean ?

Reply:
I mean placing the org.apache.* after java.* for easier search

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/587/commits/3a05f4366b7bfec8d95b5737d8acd8f4b4f2701d